src/v3/test/integration/authenticator-enroll-data-phone.test.tsx

```diff
@@ -79,6 +79,21 @@ describe('authenticator-enroll-data-phone', () => {
     );
   });
 
+  it('phone number and extension fields should be ltr even when a rtl language is set', async () => {
+    const {
+      findByTestId, user, findByLabelText,
+    } = await setup({ mockResponse, widgetOptions: { language: 'ar' } });
+
+    const phoneNumberEle = await findByTestId('authenticator.phoneNumber') as HTMLInputElement;
+
+    const methodType = await findByLabelText(/Voice call/);
```

Comment:
I see this label is still in English -- do we not actually load other languages in the tests?